tests/components/unifiprotect/sample_data/sample_nvr.json

```diff
@@ -0,0 +1,236 @@
+{
```

Comment:
Sample data should be placed in a fixtures directory in this package.

Example:
https://github.com/home-assistant/core/tree/dev/tests/components/zwave_js/fixtures